packages/config/__snapshots__/validation.spec.ts.js

```diff
@@ -158,69 +189,38 @@ exports['array of non-strings'] = {
   "type": "a string or an array of strings"
 }
 
-exports['src/validation .isNumberOrFalse returns error message when value is a not number or false 1'] = {
-  "key": "mockConfigKey",
-  "value": null,
-  "type": "a number or false"
-}
-
-exports['src/validation .isPlainObject returns error message when value is a not an object 1'] = {
-  "key": "mockConfigKey",
-  "value": 1,
-  "type": "a plain object"
-}
-
-exports['src/validation .isNumber returns error message when value is a not a number 1'] = {
-  "key": "mockConfigKey",
-  "value": "string",
-  "type": "a number"
-}
-
-exports['invalid retry value'] = {
-  "key": "mockConfigKey",
-  "value": "1",
-  "type": "a positive number or null or an object with keys \"openMode\" and \"runMode\" with values of numbers or nulls"
-}
-
-exports['invalid retry object'] = {
-  "key": "mockConfigKey",
-  "value": {
-    "fakeMode": 1
-  },
-  "type": "a positive number or null or an object with keys \"openMode\" and \"runMode\" with values of numbers or nulls"
-}
-
-exports['src/validation .isValidClientCertificatesSet returns error message for certs not passed as an array array 1'] = {
-  "key": "mockConfigKey",
-  "value": "1",
-  "type": "a positive number or null or an object with keys \"openMode\" and \"runMode\" with values of numbers or nulls"
+exports['not one of the strings error message'] = {
```

Comment:
Do we know why all of these changed? Can we copy over the old one to ensure nothing has changed here?

Reply:
I re-ordered them in the same order as before